app/recurring_job.go

```diff
@@ -875,6 +875,10 @@ func filterVolumesForJob(allowDetached bool, volumes []longhorn.Volume, filterNa
 		if util.Contains(*filterNames, volume.Name) {
 			continue
 		}
+		if volume.Status.RestoreRequired {
+			logger.Debugf("Do not create job for %v volume in restoring the backup", volume.Name)
+			continue
+		}
```

Comment:
This part is trying to prevent a volume which is restoring a backup data from taking a snapshot/backup.
`volume.Status.RestoreRequired` will be `false` after a backup restoration.

Reply:
OK, Got it